packages/constants/src/mcf.ts

```diff
@@ -438,3 +440,700 @@ export const SSEC_EQA_LIST: ISsecEqa[] = [
   {code: 'N9', description: 'Other Statement of Attainment, modular certificate or equivalent n.e.c.'},
   {code: 'XX', description: 'Not reported'},
 ];
+
```

Comment:
Since these are already in `stopwordList.js` why do we need to declare them again here?

Reply:
removed.